configure

```diff
@@ -4263,7 +4264,7 @@ fi
 ## now use all these
 PKG_CPPFLAGS="${PKG_CPPFLAGS} ${CXXFLAGS} $protobuf_cxxflags"
 
-PKG_LIBS="${PKG_LIBS} $rcpp_ldflags $protobuf_libs"
+PKG_LIBS="${PKG_LIBS} ${LDFLAGS} $rcpp_ldflags $protobuf_libs"
```

Comment:
You're right. I removed the LDFLAGS inclusion from my working copy and it looks as if `src/Makevars` is populated with the `LDFLAGS` variable correctly. I've pushed an update that removes the `LDFLAGS` so that there's no duplication.